app/services/scheduler_service.py

```
@@ -0,0 +1,266 @@
+"""
+Scheduler service for managing workflow scheduling and triggers
+"""
+
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)